portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ClientJS.testcase

```diff
@@ -0,0 +1,91 @@
+definition {
+
+	property analytics.cloud.release = "true";
+	property analytics.cloud.upstream = "true";
+	property portal.release = "false";
+	property portal.upstream = "true";
+	property property.group = "analytics.cloud.profile";
+	property testray.main.component.name = "Analytics Cloud";
+
+	setUp {
+		task ("Copy the token from AC") {
+			ACUtils.launchAC();
+
+			ACSettings.takeTokenFromAnalyticsCloud();
+		}
+
+		task ("Set up instance and sign in DXP") {
+			TestCase.setUpPortalInstance();
+
+			ACUtils.launchDXP();
+		}
+
+		task ("Add a new site and page") {
+			ACUtils.addSiteAndPage();
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		ACUtils.tearDownDXP();
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "Site Name");
+
+			CustomFields.tearDownCP();
+
+			JSONUser.tearDownNonAdminUsers();
+		}
+
+		ACUtils.tearDownAC();
+	}
+
+	@description = "Bug: LRAC-8038 | Automation ID: LRAC-11536 | Test Summary: Assert client does not send duplicated requests when the identity is the same (lowercase email string before hash)"
+	@priority = "4"
+	test CheckIdentityEventNotResendWhenEmailIsTheSame {
+		property analytics.cloud.upstream = "false";
```

Comment:
Hey @manuferreira, I think you could allow the test to run in the upstream routine. Events in DXP were not affected because of the problem classifying the user as known in the master routine was not

Reply:
Ohh nice, I thought the events were affected too. Thanks Vini